src/build/generate-contributions.ts

```diff
@@ -578,6 +578,12 @@ const chromeBaseConfigurationAttributes: ConfigurationAttributes<IChromeBaseConf
     description: refString('edge.useWebView.description'),
     default: false,
   },
+  inspectUri: {
+    type: ['string', 'null'],
+    description: refString('chrome.inspectUri.description'),
+    default:
+      '{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}', // The default value used to debug Blazor
```

Comment:
I might just leave this out since it'll be autogenerated by e.g. the C# extension generally

Reply:
You mean the whole parameter or only the default value?

Reply:
The default value

Reply:
Removed